tests/jeos/firstrun.pm

```diff
@@ -273,10 +274,24 @@ sub run {
         assert_screen 're-encrypt-finished', 600;
     }
 
-    # Skip ssh key enrollment (for now)
     unless (is_sle || is_sle_micro('<=6.0') || is_leap) {
         assert_screen 'jeos-ssh-enroll-or-not', 120;
-        send_key 'n';
```

Comment:
we cannot remove the skip option. Most of the tests will still skip it as MM option is only available for QEMU. Other hypervisors as Hyperv, VMWare, Xen wont get that feature

Reply:
ok, I see the else now! :facepalm: